subsys/fs/nvs/nvs.c

```diff
@@ -745,10 +745,10 @@ int nvs_init(struct nvs_fs *fs, const char *dev_name)
 	}
 
 	LOG_INF("%d Sectors of %d bytes", fs->sector_count, fs->sector_size);
-	LOG_INF("alloc wra: %d, %" PRIx32 "",
+	LOG_INF("alloc wra: %d, %d",
 		(fs->ate_wra >> ADDR_SECT_SHIFT),
```

Comment:
Was the switch from `%x` to `%d` intentional?  The commit message does suggest so, but the original hex representation of the mask seems more helpful.

Reply:
No it was not. I guess I should just send a new PR to fix that. Sorry.